tests/integration/test_metrics.py

```diff
@@ -1851,8 +1852,6 @@ def test_metric_bucket_encoding_dynamic_global_config_option(
             namespace: "array"
         }
 
-    print(mini_sentry.global_config)
```

Comment:
Note to self: investigate why this is not caught by automation.

Reply:
Oops, I think that was me 🤣